src/components/Availability.test.js

```diff
@@ -49,3 +49,40 @@ it("shows total slots if availability has no content", async () => {
     });
     expect(screen.queryByText("45 slots")).toBeTruthy();
 });
+
+it("shows total slots across all available days", async () => {
+    await act(async () => {
+        render(
+            <Availability
+                entry={{
+                    hasAppointments: true,
+                    appointmentData: {
+                        "10/11/2021": {
+                            hasAvailability: true,
+                            numberAvailableAppointments: 35,
+                        },
+                        "10/12/2021": {
+                            hasAvailability: true,
+                            numberAvailableAppointments: 35,
+                        },
+                    },
+                }}
+            />
+        );
+    });
+    expect(screen.queryByText("Total available: 70 slots")).toBeTruthy();
```

Comment:
Prefer `expect(getByText()).toBeVisible()` over this pattern, as it will return more descriptive errors on failure.

Reply:
Thanks for the suggestion. Just pushed the change.